R/render_email.R

```diff
@@ -0,0 +1,72 @@
+#' R Markdown render functions for the `blastula_email` output format
+#'
+#' The `render_email()` and `render_connect_email()` functions both allow for
+#' rendering an an email message. We can supply an R Markdown document (.Rmd)
+#' with the output specified as `output: blastula::blastula_email`. While the
+#' `render_email()` and `render_connect_email()` functions have similar
+#' arguments, the `render_connect_email()` is preferred when publishing to the
+#' RStudio Connect service. It allows for the inclusion of a predefined footer
+#' that contains useful links for email recipients.
+#'
+#' @param input The input file to be rendered. This should be an R Markdown
+#'   document (.Rmd) with the output specified as `output:
+#'   blastula::blastula_email`.
+#' @param envir The environment in which the code chunks are to be evaluated
+#'   during knitting (we can use `new.env()` to guarantee an empty, new
+#'   environment).
```

Comment:
When you use `new.env()`, the default parent is `parent.frame()`. So I don't think this is quite accurate. Maybe `new.env(parent = emptyenv())`, but that's a little clumsy for the `@param` docs, maybe the parenthetical should be in Details or dropped entirely.

Reply:
I'm actually in favor of dropping the extra detail entirely in this case (most people won't care and those who do would be able to figure it out anyway). I made that change.